locales/it/index.yml

```diff
@@ -3709,3 +3709,8 @@ fastLogin:
         subtitle: "Per continuare a utilizzare l’app premi su Continua la navigazione."
         submitButtonTitle: "Continua la navigazione"
         cancelButtonTitle: "Annulla"
+      transientError:
+        title: "C’è un problema temporaneo con il server"
```

Comment:
@shadowsheep1 I think here we should write the missed goal of the user: what is the implication of this error? IMHO I don't really care about the server status, nor I'm supposed to know what a server is 😆

Reply:
The transient error with Fast Login comes if the backend is down or overloaded and cannot honor the token refresh. Since we don't want to logout the user, we show this error screen inviting them to retry a bit later. But I agree with you we can found a better phrase. Any suggestion?

Reply:
aka the user tried to login and it didn't work—right?

Reply:
I would say "enter" or "access" rather then "login"? BTW, let me know!